canonical/juju-dashboard
# WD-8546 - Create secrets

## Done

- Create secrets when the add secret form is submitted.

## QA

- Go to the secrets tab in a model.
- Click the "Add secret" button.
- Fill out the form and submit.
- Refresh the page and the secret should appear in the table (the secrets will get refetched in another update).

## Details

https://warthogs.atlassian.net/browse/WD-8546
